### ISSUE TITLE
Failing requests on WeeChat 4.1.0

### ISSUE THREAD

**Issue by tardypad** [wee-slack/wee-slack#911]
I've upgraded weechat from 4.0.5 to 4.1.0 some days ago and wee-slack is now unusable.

A lot of requests are failing such as 
```
Failed requesting users.prefs.get with token xoxc-3403234350...7efc02eea0, retrying. If this persists, try increasing slack_timeout. Error (code -2): 
Failed requesting users.getPresence with token xoxc-3403234350...7efc02eea0, retrying. If this persists, try increasing slack_timeout. Error (code -2): 
Failed requesting users.info for team domain=crispnl nick=DamienTardy-Panis, retrying. If this persists, try increasing slack_timeout. Error (code -2): 
Failed requesting conversations.members for team domain=crispnl nick=DamienTardy-Panis, retrying. If this persists, try increasing slack_timeout. Error (code -2):
Failed requesting conversations.info for team domain=crispnl nick=DamienTardy-Panis, will not retry after too many failed attempts. If this persists, try increasing slack_timeout. Error (code -2):
Failed requesting conversations.history for team domain=crispnl nick=DamienTardy-Panis, will not retry after too many failed attempts. If this persists, try increasing slack_timeout. Error (code -2):
```
As suggested, I've tried to increase the timeout (from 20000 to 60000) but it doesn't help.
Downgrading back to 4.0.5 for now.

Probably it's a regression in weechat itself, but can someone else confirm the same issue?

**Comment by trygveaa** [comment]
Does it work again after downgrading to 4.0.5? Which OS/distro are you on?

Seems like at least one other person is experiencing the same: https://github.com/wee-slack/wee-slack/issues/883#issuecomment-1770139205

**Comment by trygveaa** [comment]
Ah, I can reproduce it. I'll look into it.

**Comment by tardypad** [comment]
> Does it work again after downgrading to 4.0.5? Which OS/distro are you on?
Yes it works fine on 4.0.5.
I'm using Arch Linux btw

**Comment by trygveaa** [comment]
Issue reported to WeeChat: https://github.com/weechat/weechat/issues/2033

**Comment by Ronnie76er** [comment]
I saw the other issue was fixed, so I installed weechat HEAD with brew, and it appears to be functioning properly again.

**Comment by trygveaa** [comment]
This has been fixed with WeeChat 4.1.1 which was released a few days ago.
